.markdownlint.json

```diff
@@ -1,4 +1,7 @@
 {
     "default": true,
-    "MD013": false
+    "MD013": false,
+    "MD025": {
+        "front_matter_title": ""
+    }
```

Comment:
Can I suggest you duplicate the config from CI: https://github.com/laminas/laminas-continuous-integration-action/blob/1.41.x/setup/markdownlint/markdownlint.json with the addition of the front-matter fixes - just for the sake of consistency with everything else.

Feel free to disagree… For example, all the lists will need the indentation updating etc

Reply:
It does indeed make sense to use the same rules - i did however keep the initial changes coming from MD014 (removed the dollar signs used before commands, which on click would be copied to the clipboard as well), but that can be reverted if you feel it makes it more obvious that they're console commands.

Reply:
From what I can see, all code blocks have a language association such as ` ```bash `, so removing the prompt is a good improvement as far as I'm concerned 👍